docs/pages/kbutton.vue

```diff
@@ -26,6 +26,23 @@
           <KButton text="Secondary flat button" :primary="false" appearance="flat-button" />
         </KButtonGroup>
       </DocsShow>
+      <p>
```

Comment:
I appreciate you think of documentation @josephinoo. I wonder whether we need to mention this in the docs though since many KDS components have a11 baked in, and generally from a user-developer point of view, there's no need to know the details.

@radinamatic any thoughts about this? https://deploy-preview-856--kolibri-design-system.netlify.app/kbutton/ this is the new section
![Screenshot from 2024-12-09 10-19-03](https://github.com/user-attachments/assets/78c627a7-66f8-4a0c-8072-2ba0a8e2052d)

Reply:
Thank you, @MisRob, for the feedback! You make a good point about the built-in accessibility features. I can remove the explicit documentation about ARIA attributes, since they work automatically without developer intervention. The focus should be on the component's functionality rather than its internal accessibility implementation. Let me update the PR accordingly.